llvm/include/llvm/CodeGen/MIRParser/MIParser.h

```diff
@@ -47,6 +47,7 @@ struct VRegInfo {
   } D;
   Register VReg;
   Register PreferredReg;
+  std::vector<::uint8_t> Flags;
```

Comment:
Why is the :: needed

Reply:
There's a struct named `uint8_t` defined in this class.
I believe the author meant `enum : uint8_t` instead.